strawberry/field.py

```diff
@@ -91,6 +94,9 @@ def __call__(self, resolver: _RESOLVER_TYPE) -> "StrawberryField":
         self.base_resolver = resolver
         self.type = resolver.type
 
+        self.repr = False
+        self.compare = False
+
```

Comment:
This is done in `@base_resolver.setter`. Is that not working?

Reply:
Here: https://github.com/strawberry-graphql/strawberry/blob/f14305b6b2918424f2ac70519a61617c7136f113/strawberry/field.py#L130-L132

Reply:
uh, I missed it, but no it is not working, let me check why

Reply:
oh we were missing `compare`, updated!